licensing/third-party-programs.txt

```diff
@@ -543,6 +543,30 @@ HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABI
 
 -------------------------------------------------------------
 
+14. Deformable Convolutional Networks
```

Comment:
Shouldn't we provide link to [original](https://github.com/msracver/Deformable-ConvNets) repository ? I'm no legal expert, but from my point of view "Deformable Convolutional Networks" doesn't really point to original authors for someone out of cotext.

Reply:
TPP format requires to specify name, copyright and license text.
Original repository could be mentioned in a source code that uses it. Also please make sure that this source file has corresponding copyright

Reply:
I've added mention in [here](https://github.com/openvinotoolkit/openvino/pull/5116/commits/6f2750c8508701a166d98b5d09931dcdcf5a2a3c) in #5116 . Please verify if it is correct.